biz.aQute.bndlib/src/aQute/bnd/osgi/Processor.java

```diff
@@ -1020,12 +1020,14 @@ private void doIncludes(File ubase, Properties p) {
 							} finally {
 								tmp.delete();
 							}
-						} catch (Exception mue) {
-							// ignore
+						} catch (MalformedURLException mue) {
+							if (fileMustExist)
+								error("Included file %s %s", file,
+										(file.isDirectory() ? "is directory" : "does not exist"));
+						} catch (Exception e) {
```

Comment:
We don't need the catch (Exception) block here since the catch (Exception) block below will cover this.